pylearn2/train_extensions/tests/test_roc_auc.py

```diff
@@ -173,7 +173,7 @@ def test_roc_auc_one_vs_one():
                     max_epochs: 1,
                 },
                 !obj:pylearn2.termination_criteria.MonitorBased {
-                    channel_name: train_y_roc_auc,
+                    channel_name: train_roc_auc-0v1,
```

Comment:
The original test was buggy. The written channel name doesn't exist. The issue was not provoked before, before the EpochCounter terminates the training before even MonitorBased being called.